src/configuration.model.ts

```diff
@@ -64,6 +64,24 @@ class EsploraConfig {
     batchSize: number;
 }
 
+export class BitcoinCoreConfig {
+    @IsNotEmpty()
+    @IsString()
+    rpcHost: string;
+
+    @IsNotEmpty()
+    @IsString()
+    rpcPass: string;
+
+    @IsNotEmpty()
+    @IsString()
+    rpcUser: string;
+
+    @IsNotEmpty()
+    @IsString()
+    rpcPort: string;
```

Comment:
This should be a number. Please add additional validation checks like min and max